firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -4744,6 +4744,11 @@ extension String {
 
 // MARK: - Tab Toolbar
 extension String {
+    public static let TabToolbarDataClearanceAccessibilityLabel = MZLocalizedString(
+        key: "TabToolbar.Accessibility.DataClearance.v122",
```

Comment:
Did we make the export for v122 already? Just asking since that means we need to do a special export for this to l10n? 🤔

Reply:
Yes the export was made already, this and another string (will have a PR later today for it) was recently given to us by design / product. Discussed with @adudenamedruby that these may not be translated in time, but do we want to do a special export?

Reply:
We will do a special export, yeps.

Reply:
Ok thank you 💯